spec/app_spec.rb

```diff
@@ -29,12 +29,42 @@
     expect(last_response).to be_ok
 
     json = JSON.parse(last_response.body)
-    expect(json).to eq "1.25.3"
+
+    expected_full_response = {
+      "activate.d" => false, 
+      "binary_prefix" => false, 
+      "deactivate.d" => false, 
+      "description" => "urllib3 is a powerful, sanity-friendly HTTP client for Python. Much of the Python ecosystem already uses urllib3. urllib3 brings many critical features that are missing from the Python standard libraries, such as thread safety, connection pooling, client side ssl/tls verification, support for gzip and deflate encodings, HTTP and SOCKS proxy support, helpers for retrying requests and dealing with HTTP redirects.", 
+      "dev_url" => "https://github.com/shazow/urllib3",
```

Comment:
We should send Andrey a check for every time we use his package in a test too :-p